components/sensor/teleinfo.rst

```diff
@@ -97,6 +104,15 @@ Configuration variables:
 - **uart_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the :ref:`UART Component <uart>` if you want
   to use multiple UART buses.
 
+In a teleinfo sensor, in addition to normal sensor configuration: :ref:`Sensor <config-sensor>`
+
+- **tag_name** (**Required**): Specify the tag you want to retrieve from the Teleinformation and select with what name to transmit it.
+
+In a text sensor, in addition to normal text sensor configuration: :ref:`Text Sensor <config-text_sensor>`
+
+- **tag_name** (**Required**): Specify the tag you want to retrieve from the Teleinformation and select with what name to transmit it.
```

Comment:
The name to transmit it is the sensor name itself? not the name of the tag_name, right?
```suggestion
- **tag_name** (**Required**): Specify the tag you want to retrieve from the Teleinformation.
```